acceptancetests/assess_cross_model_relations.py

```diff
@@ -205,6 +270,19 @@ def offer_endpoint(client, app_name, offer_name=None):
     return offer_url, offer_name
 
 
+def register_user_on_controller(client, username, token):
+    """Register user with `token` on `client`s controller.
+
+    :return: ModelClient object for the registered user.
+    """
+    controller_name = 'cmr_test'
+    user_client = client.clone(env=client.env.clone())
+    user_client.env.user_name = username
+    user_client.env.controller = Controller(controller_name)
+    register_user_interactively(user_client, token, controller_name)
+    return user_client
+
+
 def deploy_local_charm(client, app_name):
```

Comment:
This feels utility or otherwise not needing replicated in every test. Curious this doesn't exist somewhere.